NEW-MEMBERS.md

```diff
@@ -0,0 +1,29 @@
+## New Members
+
+The purpose of this plan is to ensure that you become familiar with the team and
+know how you will contribute. The first step is to get yourself familiar with
+our mission at [Security TAG charter](governance/charter.md).
+
+New members are advised to:
+
+* Join the [CNCF Slack team](https://slack.cncf.io/), particularly
+  [#tag-security](https://cloud-native.slack.com/messages/CDJ7MLT8S) channel and
+  introduce yourself.
+* Initially go through the following documents in the repository:
+    * [README.md](https://github.com/cncf/tag-security/blob/main/README.md)
+    * [CODE-OF-CONDUCT.md][coc]
+    * [Use cases and personas][use-cases]
+* Regularly join one of the [Zoom meetings][meeting-times] at least for the first
+  couple of months to get yourself up to speed.
+* Here are multiple ways to get involved:
+    * Join the meeting as advised above and express your area of interests or if
+      you want to work on any specific issue.
+    * Express your thoughts or ask questions on an issue you find interesting.
+    * Choose an issue where [help is
+      needed](https://github.com/cncf/tag-security/labels/help%20wanted) and
+      comment on it expressing interest.
+
+[meeting-times]: https://github.com/cncf/tag-security/blob/main/README.md#meeting-times
+[coc]: https://github.com/cncf/tag-security/blob/main/CODE-OF-CONDUCT.md
```

Comment:
nit:
```suggestion
[CoC]: https://github.com/cncf/tag-security/blob/main/CODE-OF-CONDUCT.md
```

Reply:
why capitalize reference links?  that's not visible to the reader and (for me) it makes it easier if all the "variable" are all lower case, so I don't have to remember what case was used if I want to use the reference elsewhere (unlikely in this file, but more a question of practice)

Reply:
Yes, you're right that isn't needed as it isn't visible to users. Disregard this comment then.